src/ios/NetkiSDK.m

```diff
@@ -0,0 +1,190 @@
+/********* NetkiSDK.m Cordova Plugin Implementation *******/
+
+#import <Cordova/CDV.h>
+#import <netkisdk_ios/netkisdk_ios.h>
+
+@interface NetkiSDK : CDVPlugin 
+
+- (void)configure:(CDVInvokedUrlCommand *)command;
+
+- (void)getCountryList:(CDVInvokedUrlCommand *)command;
+
+- (void)setCountry:(CDVInvokedUrlCommand *)command;
+
+- (void)getDocumentsType:(CDVInvokedUrlCommand *)command;
+
+- (void)setDocumentType:(CDVInvokedUrlCommand *)command;
+
+- (void)startIdentificationFlow:(CDVInvokedUrlCommand *)command;
+
+- (void)validateInformation:(CDVInvokedUrlCommand *)command;
+
+@end
+
+@interface NetkiSDK() <NTKCameraFlowViewControllerDelegate>
+
+@property (nonatomic, strong) NSString* callbackId;
+
+@end
+
+@implementation NetkiSDK
+
+- (void)configure:(CDVInvokedUrlCommand *)command {
+    
+    NSString *appToken = [command.arguments objectAtIndex: 0];
+    NSString *serviceCode = [command.arguments objectAtIndex: 1];
+    
+    [NetkiClient.sharedClient configureWithClientToken:appToken serviceCode:serviceCode block:^(BOOL success, NSError * _Nullable error) {
```

Comment:
Actually the serviceCode is used only for MyVeryfy app, this wont be used for this plugin, this method should be something like:
https://github.com/netkicorp/cordova-plugin-netkisdk/blob/1054e88290f59451309b121f39d2a3d81a20a970/src/android/NetkiSDK.java#L128